### DISCUSSION TITLE
buddhism/2023/eightfold/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#143] [General]
# buddhism/2023/eightfold/

https://shanenull.com/buddhism/2023/eightfold/

**Comment by shane0** [comment]
# view

https://www.accesstoinsight.org/ptf/dhamma/sacca/sacca4/samma-ditthi/index.html

**Reply by shane0** [reply]
Home
Random sutta
Random article
Abbreviations
Glossary
Index
Help!
Path to Freedom Dhamma 4 Noble Truths 4th Noble Truth
Right View
samma ditthi
© 2005
Right View is the first of the eight path factors in the Noble Eightfold Path, and belongs to the wisdom division of the path.

The definition
"And what is right view? Knowledge with regard to stress, knowledge with regard to the origination of stress, knowledge with regard to the cessation of stress, knowledge with regard to the way of practice leading to the cessation of stress: This is called right view."

— DN 22

Its relation to the other factors of the path
"And how is right view the forerunner? One discerns wrong view as wrong view, and right view as right view. This is one's right view. And what is wrong view? 'There is nothing given, nothing offered, nothing sacrificed. There is no fruit or result of good or bad actions. There is no this world, no next world, no mother, no father, no spontaneously reborn beings; no brahmans or contemplatives who, faring rightly & practicing rightly, proclaim this world & the next after having directly known & realized it for themselves.' This is wrong view...

"One tries to abandon wrong view & to enter into right view: This is one's right effort. One is mindful to abandon wrong view & to enter & remain in right view: This is one's right mindfulness. Thus these three qualities — right view, right effort, & right mindfulness — run & circle around right view."

— MN 117

The consequences of wrong view...
"In a person of wrong view, wrong resolve comes into being. In a person of wrong resolve, wrong speech. In a person of wrong speech, wrong action. In a person of wrong action, wrong livelihood. In a person of wrong livelihood, wrong effort. In a person of wrong effort, wrong mindfulness. In a person of wrong mindfulness, wrong concentration. In a person of wrong concentration, wrong knowledge. In a person of wrong knowledge, wrong release.

"This is how from wrongness comes failure, not success."

— AN 10.103

...and of right view
"When a person has right view, right resolve, right speech, right action, right livelihood, right effort, right mindfulness, right concentration, right knowledge, & right release, whatever bodily deeds he undertakes in line with that view, whatever verbal deeds... whatever mental deeds he undertakes in line with that view, whatever intentions, whatever vows, whatever determinations, whatever fabrications, all lead to what is agreeable, pleasing, charming, profitable, & easeful. Why is that? Because the view is auspicious.

"Just as when a sugar cane seed, a rice grain, or a grape seed is placed in moist soil, whatever nutriment it takes from the soil & the water, all conduces to its sweetness, tastiness, & unalloyed delectability. Why is that? Because the seed is auspicious. In the same way, when a person has right view... right release, whatever bodily deeds he undertakes in line with that view, whatever verbal deeds... whatever mental deeds he undertakes in line with that view, whatever intentions, whatever vows, whatever determinations, whatever fabrications, all lead to what is agreeable, pleasing, charming, profitable, & easeful. Why is that? Because the view is auspicious."

— AN 10.104

A thicket of wrong views
"There is the case where an uninstructed, run-of-the-mill person... does not discern what ideas are fit for attention, or what ideas are unfit for attention. This being so, he does not attend to ideas fit for attention, and attends instead to ideas unfit for attention... This is how he attends inappropriately: 'Was I in the past? Was I not in the past? What was I in the past? How was I in the past? Having been what, what was I in the past? Shall I be in the future? Shall I not be in the future? What shall I be in the future? How shall I be in the future? Having been what, what shall I be in the future?' Or else he is inwardly perplexed about the immediate present: 'Am I? Am I not? What am I? How am I? Where has this being come from? Where is it bound?'

"As he attends inappropriately in this way, one of six kinds of view arises in him: The view I have a self arises in him as true & established, or the view I have no self... or the view It is precisely by means of self that I perceive self... or the view It is precisely by means of self that I perceive not-self... or the view It is precisely by means of not-self that I perceive self arises in him as true & established, or else he has a view like this: This very self of mine — the knower that is sensitive here & there to the ripening of good & bad actions — is the self of mine that is constant, everlasting, eternal, not subject to change, and will endure as long as eternity. This is called a thicket of views, a wilderness of views, a contortion of views, a writhing of views, a fetter of views. Bound by a fetter of views, the uninstructed run-of-the-mill person is not freed from birth, aging, & death, from sorrow, lamentation, pain, distress, & despair. He is not freed, I tell you, from suffering & stress.

"The well-instructed disciple of the noble ones... discerns what ideas are fit for attention, and what ideas are unfit for attention. This being so, he does not attend to ideas unfit for attention, and attends [instead] to ideas fit for attention... He attends appropriately, This is stress... This is the origination of stress... This is the cessation of stress... This is the way leading to the cessation of stress. As he attends appropriately in this way, three fetters are abandoned in him: identity-view, doubt, and grasping at precepts & practices."

— MN 2

When one's knowledge is truly one's own
[Kaccayana:] "Lord, 'Right view, right view,' it is said. To what extent is there right view?"

[The Buddha:] "By & large, Kaccayana, this world is supported by (takes as its object) a polarity, that of existence & non-existence. But when one sees the origination of the world as it actually is with right discernment, 'non-existence' with reference to the world does not occur to one. When one sees the cessation of the world as it actually is with right discernment, 'existence' with reference to the world does not occur to one.

"By & large, Kaccayana, this world is in bondage to attachments, clingings (sustenances), & biases. But one such as this does not get involved with or cling to these attachments, clingings, fixations of awareness, biases, or obsessions; nor is he resolved on 'my self.' He has no uncertainty or doubt that, when there is arising, only stress is arising; and that when there is passing away, only stress is passing away. In this, one's knowledge is independent of others. It is to this extent, Kaccayana, that there is right view."

— SN 12.15

Abandoning the unskillful, cultivating the skillful
"Don't go by reports, by legends, by traditions, by scripture, by logical conjecture, by inference, by analogies, by agreement through pondering views, by probability, or by the thought, 'This contemplative is our teacher.' When you know for yourselves that, 'These qualities are unskillful; these qualities are blameworthy; these qualities are criticized by the wise; these qualities, when adopted & carried out, lead to harm & to suffering' — then you should abandon them...

"When you know for yourselves that, 'These qualities are skillful; these qualities are blameless; these qualities are praised by the wise; these qualities, when adopted & carried out, lead to welfare & to happiness' — then you should enter & remain in them."

— AN 3.65

See also:

The Discourse on Right View (MN 9)
Ditthi Sutta (AN 10.93) — Views
The Four Noble Truths
The Fourth Noble Truth
"Ditthi" in the General Index
Creative Commons License ©2005 Access to Insight. The text of this page ("Right View: samma ditthi", by Access to Insight) is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License. To view a copy of the license, visit http://creativecommons.org/licenses/by-nc/4.0/. Documents linked from this page may be subject to other restrictions. Last revised for Access to Insight on 30 November 2013.
How to cite this document (a suggested style): "Right View: samma ditthi", edited by Access to Insight. Access to Insight (BCBS Edition), 30 November 2013, http://www.accesstoinsight.org/ptf/dhamma/sacca/sacca4/samma-ditthi/index.html .
Help | Site map | About | Contact | Terms of use
Access to Insight is owned and managed by the Barre Center for Buddhist Studies

**Reply by shane0** [reply]
https://youtu.be/DM2couEwfT0?si=43UGntHpaLHiPK9L

**Comment by shane0** [comment]
# Right View or Right Understanding 

Right view or right understanding is the compass that points us in the right direction on our path of cultivation.
It means developing the right view about ourselves and the world, which essentially involves understanding the principle of causality, emptiness (29 kong),' and the middle way." With right understanding, we will be able to discern distorted views and futile speculations from the truth. For example, the following views reflect right understanding:
• Understanding that seeking sensual pleasures and attempting to satiate the ego will lead to suffering instead of true, lasting happiness.
• Understanding that self-centeredness will eventually bring adversity into own our lives, while benefiting others is to benefit ourselves, and vice versa.
• Realizing that all phenomena are empty in nature; they do not have an independent existence or inherent, fixed charac-teristics.
• Seeing all things and all beings as interdependent and inter-connected.
• Realizing that all beings are essentially equal in their original
nature.

**Reply by shane0** [reply]
The twelve links of interdependence: the chain of causes and condlions that lead to the cycle of rebirth, or repeated patterns of thoughtand behavior. This teaching will be taught in Level 3.

Emptiness: an important Buddhist teaching that shows all phenomena, including what we perceive as our "self" are without an independent existence or inherent, fixed characteristics; therefore all things are imper-manent, changing, and interdependent.


The middle way: an important Buddhist teaching that describes reality as it is, free of dualistic views. Emptiness and the middle way will be discussed in more detail in Level 3.


To purify our verbal action, we avoid false speech. To purify our mental action, we refrain from preoccupying our mind with thoughts of craving, anger, self-gratification, or ignorance.
The Buddha embodies the enlightened mind, our pure awareness. The Dharma embodies the truth. The Sangha represents our intrinsic purity
and harmony.
